docs/patterns/logo-wall/examples.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Examples
+heading: Logo wall
+sidenavTitle: Logo wall
+layout: layouts/pages/pattern.njk
+hasToc: true
+order: 20
+tags:
+  - logowallPatterns
+subnav:
+  collection: logowallPatterns
+  order: 4
+importElements:
+  - rh-cta
+  - rh-surface
+---
+
+<script type="module">
+  import '@rhds/elements/lib/elements/rh-context-picker/rh-context-picker.js';
+</script>
+
+<link rel="stylesheet" href="/styles/samp.css">
+
+<link rel="stylesheet" href="{{ '../logo-wall-lightdom.css' | url }}" />
+
+## Within a promo band (bordered)
+  {% include './patterns/1x1-grid.html' %}
+
+## Within a promo band (borderless)
+  {% include './patterns/1x1-grid-flat.html' %}
```

Comment:
It seems like there might be a bug for these two. At about 1200px to 1520px, the content breaks out of the container.

Reply:
I could reproduce this in chrome, but not in firefox. At first it seemed like it might be a layout issue with `<uxdot-pattern>`, but I tried replacing that element with just the pattern code, and it looks like indeed the non-wrapping examples do break out of their container when the content is too wide for it.

Reply:
https://github.com/RedHat-UX/red-hat-design-system/issues/1782